tests/unit/main.c

```diff
@@ -399,20 +407,30 @@ ctest_is_unit_test(const char *argv0)
    return 0;
 }
 
-// Determine the log message verbosity to use for this test run.
-// If env var VERBOSE is unset, or is set to "0" or "false", the be quiet.
-// Otherwise, be verbose.
-static bool
+/*
+ * Determine the log message verbosity level to use for this test run.
+ *
+ * If env var VERBOSE is unset, or is set to "0" then be quiet.
+ * If env var VERBOSE=3 then be verbose reporting only errors.
+ * Otherwise, settle for the specified verbosity level.
+ *
+ * NOTE: Currently, the only other level we support is level=7 to print all
+ *       debugging output.
```

Comment:
Revise this comment line, since other values are used and are ok.

Reply:
I just deleted this NOTE: line as trying to improve it got harder than the value added.